common/views/components/BasePage/EventPage.js

```diff
@@ -93,9 +93,24 @@ const EventPage = ({ event }: Props) => {
   /* https://github.com/facebook/flow/issues/2405 */
   /* $FlowFixMe */
   const FeaturedMedia = event.promo && <UiImage tasl={tasl} {...image} />;
-  const formatTag = event.format ? [{text: event.format.title}] : [];
+  const formatTag = event.format ? [{text: event.format.title + audiencesString(event.audiences)}] : [];
   const interpretationsTags = event.interpretations ? event.interpretations.map(i => ({text: i.interpretationType.title})) : [];
   const TagBar = <Tags tags={formatTag.concat(interpretationsTags)} />;
+  function audiencesString(audiences) {
+    if (audiences.length > 0) {
```

Comment:
Do you need this `if/else`? If audiences is an empty array, then wouldn't the `reduce` give you back the `''`' you put in as the initial argument?